### PR TITLE
Expose the supported metadata formats

### PR DESCRIPTION
**Pull request by slesaad**
# Changes proposed

- Expose supported metadata formats by providing a function `pyQuARC.supported_formats()`
- Rename `get_version()` to `version()`

# Screenshot

image

### CODE REVIEW

File: pyQuARC/__init__.py [hunk 1]
```diff
@@ -15,5 +17,8 @@
 with open(f"{ABS_PATH}/version.txt") as version_file:
     __version__ = version_file.read().strip()
 
-def get_version():
+def version():
     return __version__
+
+def supported_formats():
```
Comment (CarsonDavis) [inline pyQuARC/__init__.py:23]: Can we add a short description about what the formats are?

Comment (CarsonDavis) [reply pyQuARC/__init__.py:23]: maybe just a link to some other documentation

Comment (slesaad) [reply pyQuARC/__init__.py:25]: done

**Comment by xhagrg** [comment]
why not enums or constants? something like: `pyQuARC.FORMATS`?

File: pyQuARC/__init__.py [hunk 2]
```diff
@@ -15,5 +17,24 @@
...
+    """
     return __version__
+
+def supported_formats():
+    """Returns a list of metadata formats supported by pyQuARC.
+       One of the formats in the list can be passed to ARC.validate()
+       Default is `echo-c`.
+
+       "dif10": Dif10 collectioin metadata format
+       "echo-c": Echo10 collection metadata format
```
Comment (code-geek) [inline pyQuARC/__init__.py:31]: Nitpick: why "dif10" but not "echo10-c"?

Comment (xhagrg) [reply pyQuARC/__init__.py:31]: I assume these are formats followed by CMR.

**Comment by slesaad** [comment]
@xhagrg i was debating between that vs a function; you think that's better?

File: pyQuARC/__init__.py (hunk 2 above)
Comment (slesaad) [reply pyQuARC/__init__.py:31]: `umm-c` and `umm-g` are actually standards in CMR, but ECHO formats are just called ECHO collection/granule.

there's `dif` and `dif10` both, but only `echo10`.

so i guess, i was trying to make it the standard way of specifying collection or granule metadata by using `echo-c` and `echo-g` taking inspiration from `umm-c` and `umm-g`. Plus, `echo10-c` is just longer and harder to write than `echo-c`

**Comment by xhagrg** [comment]
Version is reading from a file, hence a method. These are hard coded, hence a ENUM/constant.

**Comment by slesaad** [comment]
does that matter for the user though since we're exposing it to the users? user doesn't care what's going on behind the hood. we can just as easily make `version()` into a constant.

not that i have specific preference, just trying to understand why the preference

**Comment by xhagrg** [comment]
version is reading from a file. these lists stored in constants?

**Comment by xhagrg** [comment]
LGTM.

**Pull request merged**